docs/BHoM_Adapter/Adapter-Actions---advanced-parameters.md

```diff
@@ -1,6 +1,6 @@
 # Adapter Actions: advanced parameters 
 
-The [Adapter Actions](/documentation/BHoM_Adapter/Adapter-Actions) have some optional inputs that allow to refine their behaviour.
+The [Adapter Actions](Adapter-Actions.md) have some optional inputs that allow to refine their behaviour.
```

Comment:
Thanks for changing this and other links like this. I see you've changed several other links in this and other files to be consistent with the strategy of #86. I guess that you've been guided by the Link Checker errors.

Just as a note, I noticed that, for some reason, the Markdown link checker doesn't detect the links on Line 7 in this same file as "wrong" (although they aren't wrong, they're just in a form that the link checker isn't supposed to like). I tried modifying those links with some gibberish at the end (e.g. `[Introduction to BHoM_Adapter](/documentation/BHoM_Adapter123)` and the link checker still fails to detect them as wrong.

However, we don't need to worry about these kind of links in this PR -- I'm just highlighting this as a problem to be solved when doing the link uniformation as part of #86 . 

I think that a simple workaround to ensure we are being consistent can be, I think, to search for all occurrences of this pattern `](/` in the text, because it identifies absolute links (i.e. referenced from the root; and because MkDocs and VSCode disagree on the right root, they aren't compatible with both, as we noticed in #86). Here's an example of this search result, which indeed seem to confirm that this strategy can work:

![image](https://github.com/BHoM/documentation/assets/6352844/02f4bf57-5bb6-448d-9327-3d031c0ef0c4)

Reply:
Thanks for this also @alelom - yes - makes sense.
Once this PR has gone in - will be good to finalise all those tips and flush through as you say.

Reply:
> (This lengthy comment is **not** a request for changes! 😄 )

I appreciated this clarification!! I must admit when I first saw your block of text - I did think! _Gulp! What have I done!_ 😆